content_scripts/setupMenuBar.js

```diff
@@ -99,6 +101,9 @@ createNameSpace('realityEditor.gui');
 
         const toggleFlyMode = new MenuItem(ITEM.ToggleFlyMode, { toggle: true, shortcutKey: 'F', defaultVal: false }, null);
         menuBar.addItemToMenu(MENU.Camera, toggleFlyMode);
+        
+        const focusCamera = new MenuItem(ITEM.FocusCamera, { shortcutKey: 'G' }, null);
+        menuBar.addItemToMenu(MENU.Camera, focusCamera);
```

Comment:
I'm a little bit double-minded about this one. On one hand, it's nice to have it visible in the menu so you know that the G keyboard shortcut exists. On the other hand, if you click on the menu item and the mesh isn't underneath your mouse position when you focus, it seems to focus onto nothing (just seeing a grey screen).